### DISCUSSION TITLE
Difference between ookii-dialogs and ookii-dialogs-wpf?

### DISCUSSION THREAD

**Discussion by ultimate-tester** [ookii-dialogs/ookii-dialogs-wpf#15] [Q&A] [answered]
Hi,

As a first time user of ookii-dialogs, I'm wondering what's up with the different packages with different authors and sites.
Is this a copy of ookii-dialogs or a continuation of it? Which exactly is the "real" one?

**Comment by augustoproiete** [comment]
Hi @ultimate-tester, what are the different packages with different authors and sites that you are referring to?

The Ookii Dialogs project (both for WPF and WinForms) was originally created by [Sven Groot](http://www.ookii.org) in 2009, and it's being maintained by me since 2018 on these repos below:

- https://github.com/augustoproiete/ookii-dialogs-wpf
- https://github.com/augustoproiete/ookii-dialogs-winforms

The NuGet packages are, respectively:

- https://www.nuget.org/packages/Ookii.Dialogs.Wpf
- https://www.nuget.org/packages/Ookii.Dialogs.WinForms

**Comment by ultimate-tester** [comment]
Hi augustoproiete,

Thanks for the quick response. So there's this package called "Ookii.Dialogs" and as I now understand the packages "Ookii.Dialogs.Wpf" and "Ooki.Dialogs.WinForms" are its succesors? So the "Ookii.Dialogs" is deprecated? It was just a little confusing when selecting packages, seeing a more popular (but older) package. I was just curious if the ".Wpf" version was a rip-off or an official version, but now I understand I should use the ".Wpf" version and not look at "Ookii.Dialogs" anymore. Thanks!

**Comment by augustoproiete** [comment, marked as answer]
Got it :relaxed:. Thanks for bringing this to my attention.

Indeed, the NuGet package [`Ookii.Dialogs`](https://www.nuget.org/packages/Ookii.Dialogs/) is deprecated. It was published by [superkinhluan](https://www.nuget.org/profiles/superkinhluan) back in 2011 from the original code from Sven at that time and it accumulated a number of downloads over the years whilst [`Ookii.Dialogs.Wpf`](https://www.nuget.org/packages/Ookii.Dialogs.Wpf/) was only published years later in 2018 which is why it has a smaller number of downloads.

I went to NuGet to search for packages with the same name and here is a summary:

| | Old/deprecated package | | New package you should use |
| --- | --- | --- | --- |
| :package: | [Ookii.Dialogs](https://www.nuget.org/packages/Ookii.Dialogs/) | :arrow_right: | **[Ookii.Dialogs.Wpf](https://www.nuget.org/packages/Ookii.Dialogs.Wpf/)** v1.1.0 or higher |
| :package: | [Ookii.Dialogs.Wpf.NETCore](https://www.nuget.org/packages/Ookii.Dialogs.Wpf.NETCore/) | :arrow_right: | **[Ookii.Dialogs.Wpf](https://www.nuget.org/packages/Ookii.Dialogs.Wpf/)** v2.0.0\* or higher
| :package: | [Ookii.Dialogs.WindowsForms](https://www.nuget.org/packages/Ookii.Dialogs.WindowsForms/) | :arrow_right: | **[Ookii.Dialogs.WinForms](https://www.nuget.org/packages/Ookii.Dialogs.WinForms/)** v1.1.0 or higher |

(\*) Not yet published in nuget.org. Will be published in the next few days.

---

ps: I reached out to [superkinhluan](https://www.nuget.org/profiles/superkinhluan) earlier today and he was nice enough to transfer the ownership of his [`Ookii.Dialogs`](https://www.nuget.org/packages/Ookii.Dialogs/) package to me, so that I could mark it as deprecated, which should help with the confusion moving forward.

[![image](https://user-images.githubusercontent.com/177608/90265040-61421400-de28-11ea-8239-36fd8e0c7b3f.png)](https://www.nuget.org/packages/Ookii.Dialogs/)

**Comment by ultimate-tester** [comment]
Thanks for the quick replies and action! Closed!
